### ISSUE TITLE
Weird appearance on PhpStorm 2020.3

### ISSUE THREAD

**Issue by meichengg** [elliotwaite/jetbrains-cyberpunk-theme#1]
Hi, i love your theme. Unfortunately, it appears very weird on my IDE, uglier i believe for both color scheme and fonts, i'll attach some photos below.

My IDE specs:
- PhpStorm 2020.3
- macOS 10.15.7
- JDK 11.0.9

Btw, may i know what version of JDK and IDE version you are using? Because i saw some topics mentioning difference between JDK 8 and 11 causing appearance and font troubles.

Some pics:
![image](https://user-images.githubusercontent.com/25204246/103072825-ecae8b80-45f8-11eb-8869-158d3fb833a4.png)
![image](https://user-images.githubusercontent.com/25204246/103072830-f506c680-45f8-11eb-9f0c-5ac20e37e0da.png)

**Comment by elliotwaite** [comment]
Stange, I'm not sure what could be causing this. The theme only contains color settings so it shouldn't affect the font. But I'm not sure why the colors look off. I'm using IntelliJ IDEA 2020.2.2 (community edition) to build the plugin.

**Comment by meichengg** [comment]
For the IDE font, i had to use Verdana font because the default font on macOS looks pretty ugly and thin. That's the only custom setting i had. Can you try to test the plugin on PhpStorm or IDEA Pro?

![image](https://user-images.githubusercontent.com/25204246/103164038-b2333180-4838-11eb-95da-59df2f90be9a.png)

**Comment by elliotwaite** [comment]
I currently don't have access to their non-free IDEs, but I just made a request to JetBrains through their [Free License Program](https://www.jetbrains.com/community/opensource/#support) to see if I could get access to these paid IDEs to test out this plugin. I'll let you know when I hear back from them.

**Comment by meichengg** [comment]
You can try 30-day trial for all of Jetbrains products i believe, as i'm using the trial too 😛

**Comment by elliotwaite** [comment]
I think I'll save the free trial since I might need it for work in the near future.

**Comment by meichengg** [comment]
Hmm, then there is a trial reset plugin if you want to test, can you give it a shot?

**Comment by meichengg** [comment]
Cuz i can't find any other themes that satisfy me 😭

**Comment by elliotwaite** [comment]
If you go into `Preferences > Editor > Color Scheme`, do you have the `Cyberpunk` scheme selected? And if so, when you check out the colors used, for example in `Preferences > Editor > Color Scheme > Language Defaults`, do the colors used match the colors in the palette listed here: https://github.com/elliotwaite/jetbrains-cyberpunk-theme/blob/master/src/main/resources/cyberpunk.theme.json

If any of them are off, you can try manually changing them to the colors in the color palette. And you can make specific changes for the PHP syntax by looking in the PHP section (I think, I don't have a PHP section in my settings since my IDE doesn't support PHP).

**Comment by huyndnse63331** [comment]
Is there any way that i can export the color scheme settings to JSON so i can compare them easier?

**Comment by elliotwaite** [comment]
You can also view the colors in the color scheme by going to `Preferences > Editor > Color Scheme` and clicking the gear icon to the right of the scheme dropdown menu and then selecting `Export > IntelliJ IDEA color scheme (.icls)`. And then you can compare the colors in that exported file to this file: https://github.com/elliotwaite/jetbrains-cyberpunk-theme/blob/master/src/main/resources/cyberpunk.xml

**Comment by meichengg** [comment]
I just did, they were completely matching.

**Comment by elliotwaite** [comment]
Hmm, strange. I'm not sure what is causing the issue then.

**Comment by meichengg** [comment]
Unless you give a try on Pro editions. By the way, can i ask for your IDE font settings and color scheme font settings too?

**Comment by elliotwaite** [comment]
For the font, try this:
* `Preferences > Editor > Font > Font`: Source Code Pro (downloadable [here](https://fonts.google.com/specimen/Source+Code+Pro))
* `Preferences > Appearance & Behavior > Appearance > Antialiasing > IDE`: Greyscale
* `Preferences > Appearance & Behavior > Appearance > Antialiasing > Editor`: Greyscale

**Comment by meichengg** [comment]
What about this?

![image](https://user-images.githubusercontent.com/25204246/103328696-755f7880-4a8c-11eb-8029-c8937380cdf4.png)

**Comment by elliotwaite** [comment]
I use the font "Source Code Pro" at a font size of 14.

**Comment by elliotwaite** [comment]
For that, I have it unchecked. I use the standard font they have set for macOS, but I'm not sure what it is.

**Comment by meichengg** [comment]
Can you tell me which JDK version are you using for the IDE?

**Comment by elliotwaite** [comment]
I build the plugin using Gradle, and my Gradle JVM is: java version "1.8.0_222"

**Comment by meichengg** [comment]
I think i may know what the problem is. Can you try to switch to JDK 11.0.9 and reopen the IDE, you may get font issues same as mine, and can you tell me you are using IDE-bundled JDK 8 or which vendor did you get the JDK 8 (oracle, openjdk, ...)?

**Comment by elliotwaite** [comment]
Oh, the runtime my IDE is using is `11.0.9.1+11-b1145.63`. I'm not noticing any font issues while using it with the Cyberpunk theme. The theme shouldn't affect the font, do any other themes you install affect the font?

**Comment by elliotwaite** [comment]
Good news. JetBrains approved my request for the open source license. I tested out the theme on PhpStorm and fixed those purple colors. They were part of the default color scheme for PHP files. I made those items that were purple now default of the standard colors, which ended up making most of those syntax parts switch from purple to the light-grey/white color. The changes were included in the latest update (version 1.0.4). Let me know if this resolves the issue.
